reconcile/utils/terrascript_aws_client.py

```diff
@@ -2451,6 +2455,42 @@ def populate_tf_resource_s3_cloudfront(self, resource, namespace_info):
             '${' + cf_oai_tf_resource.id + '}'
         tf_resources.append(Output(output_name_0_13, value=output_value))
 
+        # aws_s3_bucket_acl
+        values = common_values.get('distribution_config', {})
+        if 'logging_config' in values.keys():
+            logging_config_bucket = values['logging_config']
+            values = {}
+            values['name'] = identifier
+            values['tags'] = common_values['tags']
+
+            access_control_list = {
+                'grant': {
+                    'grantee': {
+                        'id': '${' + data.aws_canonical_user_id.id + '}',
+                        'type': 'CanonicalUser',
+                    },
+                    'permission': 'FULL_CONTROL',
+                },
+                'owner': {
+                    'id': '${' + data.aws_canonical_user_id.id + '}',
+                }
+            }
+
+            values['bucket'] = logging_config_bucket.get('bucket')
```

Comment:
the output of `logging_config_bucket.get('bucket')` is the bucket url fully-qualified hostname. We need to get only the first part